### DISCUSSION TITLE
New BS5 Accordion: number of items restricted?

### DISCUSSION THREAD

**Discussion by chess-server** [twbs/bootstrap#37096] [Q&A]
In Bootstrap 4 I am using an accordion with approximately 50 entries (a list of IP Addresses and their details). Clicking on one button opens the details and closes any open ones. This works as well in Bootstrap 5 with the new accordion, but only for upto 20 buttons. When there are more entries, then one has to click twice on the button to open the item. Is this restricted number intended or just a too small array?

**Comment by mdo** [comment]
I don't think there are any restrictions on this—thoughts, @GeoSot?

**Comment by GeoSot** [comment]
I don't think either, but a codepen example could help to identify possible glitch
